archiver/archiver_test.go

```diff
@@ -70,20 +69,11 @@ func TestJobsArchival(t *testing.T) {
 		t.Setenv("JOBS_DB_PASSWORD", postgresResource.Password)
 	}
 
-	jobsdb.Init()
 	misc.Init()
-	jd := &jobsdb.HandleT{
-		TriggerAddNewDS: func() <-chan time.Time {
-			return make(chan time.Time)
-		},
-	}
-	require.NoError(t, jd.Setup(
-		jobsdb.ReadWrite,
-		false,
+	jd := jobsdb.NewForReadWrite(
 		"gw",
-		[]prebackup.Handler{},
-		nil,
-	))
+		jobsdb.WithDBHandle(postgresResource.DB),
```

Comment:
What's the point of `t.Setenv` above? Would it be better to use `WithConfig` instead of `WithDBHandle`? Supporting both seems like an overkill to me

Reply:
tried c.Set("DB.host") etc. but the `GetConnectionString` uses default config handle, so `c.Set("DB.host")` is doesn't work.
In all other tests, we either use `t.Setenv("JOBS_DB_HOST", postgresContainer.Host)` or `config.Set("DB.name", postgresContainer.Database)`(default global config).

`GetConnectionString` is used in a lot of places without respective handle specific config.
So going ahead with `t.Setenv` for now.